lessons/lesson-template.md

```diff
@@ -0,0 +1,74 @@
+- title: 
+- tags: 
+
+###Standards:
+(Overarching learning goals, e.g. 'Students become comfortable with Angular')
+
+### Objectives
+(Concrete, measurable tasks and learnings that map back to the overarching learning goals)
+Eg. By the end of the lesson, students will be able to:
+* Facilitate a daily standup
+* Define sprints and minimum viable products
+
+### Schedule
+* Warmup: link to challenge problem
+* Morning lecture: e.g. Intro to Arrays
+* Morning exercises: Link to exercises
+* Midday checking: link to checking
+* Lunch
+* Afternoon lecture: e.g. More Arrays
+* Afternoon exercises: Link to exercises
+* Review
+
+### Resources
+Add links for slides, live coding examples, helpful resources, etc.
+
+###Assessment Materials
+- Midday Check-in and solutions (add links)
+- Exercises and solutions (add links)
+- Homework and solutions (add links)
+
```

Comment:
What's the workflow for solutions to exercises and homework? Do we post these along with the questions, or post them later on? There's something philosophically pure about posting the solutions along with the assignments, but I think for some the temptation to peek will be too great.